src/update.js

```diff
@@ -0,0 +1,31 @@
+export const initStatusUpdates = (
+  tasksArray,
+  saveToLocalStorage,
+  clearListButton,
+  updateIndexes,
+  performJob,
+  saveTasksToLocalStorage
+) => {
+  const checkboxes = document.querySelectorAll("input[type='checkbox']");
+  console.log(checkboxes);
+  checkboxes.forEach((taskItem) => {
+    const checkbox = taskItem.querySelector("input[type='checkbox']");
+    checkbox.addEventListener("change", () => {
+      const taskIndex = parseInt(taskItem.getAttribute("data-index")) - 1;
+      checkboxes[taskIndex].completed = checkbox.checked;
+      saveTasksToLocalStorage();
+      performJob();
+    });
+  });
+
+  clearListButton.addEventListener("click", () => {
+    tasksArray = tasksArray.filter((task) => !task.completed);
+    saveToLocalStorage();
+  });
+
+  const savedTasks = JSON.parse(localStorage.getItem("tasks"));
+  tasksArray = savedTasks || [];
+
+  updateIndexes();
+  performJob();
+};
```

Comment:
- [ ] You have done a great job storing the to-do tasks in `localStorage` 👏 However, when the page gets reloaded, the `to-do` application doesn't show the list of `to-do`s in `localStorage`. Please fix this issue 🙏 
![image](https://github.com/efmomoh/Webpack-demo/assets/46839548/162b6807-00e9-42cd-8616-fc832b371f88)

Reply:
This change has been done well